packages/web-api/src/WebClient.ts

```diff
@@ -154,7 +161,10 @@ export class WebClient extends Methods {
     }
 
     const response = await this.makeRequest(method, Object.assign(
-      { token: this.token },
+      {
+        token: this.token,
+        team_id: this.teamId,
```

Comment:
This means all web API calls have this `team_id` parameter in the case of org-level installations, right? My understanding is that the server-side just ignores the param(it doesn't return any errors/warnings about it) when it doesn't need it. On this assumption, this looks good to me.

Reply:
yeah, we pretty much always pass `team_id` when making api calls if `team_id` is available in the `clientOptions`. server-side will ignore the param and won't give any errors or warnings.